### ISSUE TITLE
Can the server be deployed with high availability?

### ISSUE THREAD

**Issue by Claude-Zq** [sergelogvinov/helm-charts#14]
Currently, helm-chart only supports the deployment of a single server. Can multiple servers be deployed to achieve high availability?
https://www.jetbrains.com/help/teamcity/2024.07/multinode-setup.html?TeamCity%20Documentation

**Comment by sergelogvinov** [comment]
Hi, thank you for this information. I've been using TeamCity for more than six years and thought it was a single-daemon setup.

Yes, it makes sense to set it up as a high-availability solution... 👍

**Comment by Claude-Zq** [comment]
Will this helm-chart support high-availability deployment in the future?This is the only official helm I've found.
I found another helm-chart that supports HA, but it's not very active and, you know, unofficial.
https://artifacthub.io/packages/helm/teamcity-server/teamcity-server

**Comment by sergelogvinov** [comment]
Yes, I will add it to my list to implement. I have just one concern – it uses a shared folder between daemons, which could be a single point of failure. I don’t recall any downtime with TS, except during the upgrade process.
 
My chart is also unofficial, I do not work at jetbrains...

**Comment by sergelogvinov** [comment]
Hi,

My first idea was to use first pod-0 to the humans requests (ingress), all others for the workers.
But if you have a better idea, I'm open to suggestions.

TeamCity ha currently implemented as an alpha version. There are many things to improve here )
